test/Conversion/TorchToLinalg/elementwise.mlir

```diff
@@ -84,3 +84,28 @@ func @elementwise$static_1(%arg0: !torch.vtensor<[?],f32>, %arg1: !torch.vtensor
   %1 = torch.aten.mul.Tensor %arg0, %arg1 : !torch.vtensor<[?],f32>, !torch.vtensor<[1],f32> -> !torch.vtensor<[?],f32>
   return %1 : !torch.vtensor<[?],f32>
 }
+
+// CHECK-LABEL: func @elementwise_toDtype
```

Comment:
I don't think we need this test.  There is not any specific new code logic related to elementwise ops. And the operation itself is covered by e2e test already.